index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Hydrant</title>
     <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/hydrant.svg" />
```

Comment:
If anyone has thoughts on changing this, lmk! I just put this here cause I think we should have a favicon in general (fixes #31) but we can replace this once @tangluna makes some assets :)